app/src/main/java/co/develhope/meteoapp/DataSearches.kt

```diff
@@ -0,0 +1,3 @@
+package co.develhope.meteoapp
+
+data class DataSearches(val recentCitySearch: String, val recentTemperatureSearch: String, val recentWeatherSearch: String)
```

Comment:
Qui mancano sicuramente dei dati per poi far ele chiamate di rete, tienilo presente per il prossimo futuro

Reply:
Il nome è un pò fuorviante, sarebbe megli ocambiarlo in qualcosa di più parlante, vediamo insieme